coredns/datadog_checks/coredns/coredns.py

```diff
@@ -0,0 +1,108 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.checks.prometheus import PrometheusCheck
+from datadog_checks.errors import CheckException
+
+
+class CoreDNSCheck(PrometheusCheck):
+    """
+    Collect CoreDNS metrics from Prometheus
+    """
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        super(CoreDNSCheck, self).__init__(name, init_config, agentConfig, instances)
+        self.NAMESPACE = 'coredns'
+
+        self.metrics_mapper = {
```

Comment:
can we make this configurable in the yaml? making this the default and then merging a yaml metrics_mapper map with this one would be a good idea I think